docs/content/en/docs/yaml-crd-ref/config.md

```diff
@@ -26,33 +26,50 @@ spec:
    Must be set to `KeptnConfig`.`
 
 * **metadata**
-  * **name** -- Unique name of this collector.
-    This is the name that identifies this collector
-    in the `KeptnConfigList` CR.
-    * Must be an alphanumeric string and, by convention, is all lowercase.
-    * Can include the special characters `_`, `-`, (others?)
-    * Should not include spaces.
+  * **name** -- Unique name of this set of configurations.
+    Names must comply with the
+    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
+    specification.
 
-## Usage
+* **spec**
+  * **oTelCollectorUrl** -- The URL and port of the OpenTelemtry collector
+  * **keptnAppCreationRequestTimeoutSeconds** --
+    interval in which automatic app discovery searches for workloads
+    to put into the same auto-generated [KeptnApp](app.md).
+
+Each [KeptnApp](app.md) should have a single `KeptnConfig` CRD
+that describes all configurations for that application.
 
-The `KeptnConfigList` CR is generated automatically
-and contains a list of all valid `KeptnConfig` CR's.
+## Usage
 
 ## Example
 
+### oTel example
+
+This example specifies the URL of the OpenTelemetry collector
+and the interval for
+running the automatic app discovery:
+
 ```yaml
-apiVersion: options.keptn.sh/v1alpha1
+apiVersion: options.keptn.sh/v1alpha2
 kind: KeptnConfig
 metadata:
-  name: keptnconfig-sample
+  name: otel-url
```

Comment:
I would name this something like `klt-config` to make it clear that this is for KLT in general.
```suggestion
  name: klt-config
```

Reply:
done